### ISSUE TITLE
Can you give us the possibility to reuse name ioBroker.homekit

### ISSUE THREAD

**Issue by GermanBluefox** [husky-koglhof/ioBroker.homekit#12]
I want to write the new adapter for homekit, but the good name is occupied.
Can you give us the rights to overwrite the package on npm?

`npm owner add bluefox iobroker.homekit`
Thank you.

**Comment by husky-koglhof** [comment]
Hi, I've invited you as an admin.
I'm no longer developing this adapter cause I'm working on a different branch.

**Comment by Apollon77** [comment]
What you mean with different branch? Do you plan to bring out a new version?

**Comment by GermanBluefox** [comment]
Thank you.

вс, 5 авг. 2018 г., 8:20 Ingo Fischer <notifications@github.com>:

**Comment by husky-koglhof** [comment]
The last time I spent with developing an app, which allows you to configure ioBroker adapters natively from Android and iOS. The app is not ready for show yet.
The self-development of the zWave adapter or other adapter is currently on hold, I just do not have the time.
Homekit itself is great, but I have the problem that I also develop iOS apps and thus have access to data that I may not use in Open Source.

**Comment by foxriver76** [comment]
@GermanBluefox Now that you have access, do you plan to do any further development on the basis of this adapter or yahka? If yes, on which one and would you like to create a repo for it, where devs can make PR's to it?

**Comment by GermanBluefox** [comment]
I wanted to create from scratch the homekit... With usage of the new types-detector (https://github.com/ioBroker/ioBroker.type-detector). But I don't know if I find time till Silvester for it.

**Comment by GermanBluefox** [comment]
Can you help here? What is your plan?

**Comment by foxriver76** [comment]
From scratch can be very time consuming. I also don't know if I have enough time to build something like this in the next few months. Also I'm not that much in the logic of Apple Homekit. I thought if your intention was to further develop one of the existing adapters, I could fix some minor things, like screwed up logging of yahka. But if someone makes a start from scratch I could also sometimes find time to contribute to. :)

**Comment by GermanBluefox** [comment]
Is it not better to use telegram for communication? :)

**Comment by foxriver76** [comment]
Yes, we can do so. How can I find you there? You can write me your contact info via PM in iobroker forum: foxriver76

**Comment by GermanBluefox** [comment]
I added you into iobroker-dev channel. And I am there too. :=)

**Comment by foxriver76** [comment]
Where did you add/invite me? I didn't get any notification. @GermanBluefox

**Comment by GermanBluefox** [comment]
@foxriver76 what is your telegram nick? I was confused with foxthefox...
And thought that I have invited you. :)
Or write me at info at iobroker.net

сб, 15 сент. 2018 г., 23:53 Max Hauser <notifications@github.com>:

**Comment by foxriver76** [comment]
It‘s foxriver76 now. I had not set any before.
